extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/AbstractDataSourceProducer.java

```diff
@@ -143,8 +143,12 @@ public AgroalDataSource createDataSource(String dataSourceName,
                 agroalConnectionConfigurerHandle.get().disableSslSupport(dataSourceBuildTimeConfig.dbKind.get(),
                         dataSourceConfiguration);
             } else {
-                log.warnv("Agroal does not support disabling SSL for database kind {0}",
-                        dataSourceBuildTimeConfig.dbKind.get());
```

Comment:
That's not the right fix. You should just use `resolvedDbKind` here instead of the one from the config.

Reply:
I wasn't aware of that, thanks

Reply:
We have this layer for compatibility reason as the kind might not come from the config.